File: second/helloWorld.php
<?php
echo "Hello, World!";
echo "\nWelcome to PHP programming!";
echo "\nLet's sort some arrays.";

// Sorting an array of numbers
$numbers = array(4, 2, 8, 6);
sort($numbers);
echo "\nSorted numbers: ";
print_r($numbers);

// Sorting an array of strings
$strings = array("banana", "apple", "cherry");
sort($strings);
echo "\nSorted strings: ";
print_r($strings);
?>

<!-- Hello, World!
Welcome to PHP programming!
Let's sort some arrays.
Sorted numbers: Array ( [0] => 2 [1] => 4 [2] => 6 [3] => 8 )
Sorted strings: Array ( [0] => apple [1] => banana [2] => cherry )
-->

File: second/helloWorld.kt
fun main() {
    println("Hello, World!")
    println("Welcome to Kotlin programming!")
    
    val numbers = arrayOf(5, 2, 9, 1, 5, 6)
    println("Original array: ${numbers.joinToString(", ")}")
    
    bubbleSort(numbers)
    println("Sorted array using Bubble Sort: ${numbers.joinToString(", ")}")
    
    val numbers2 = arrayOf(3, 8, 2, 5, 1, 4)
    println("Original array: ${numbers2.joinToString(", ")}")
    
    selectionSort(numbers2)
    println("Sorted array using Selection Sort: ${numbers2.joinToString(", ")}")
}

fun bubbleSort(arr: Array<Int>) {
    val n = arr.size
    for (i in 0 until n-1) {
        for (j in 0 until n-i-1) {
            if (arr[j] > arr[j+1]) {
                val temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
}

fun selectionSort(arr: Array<Int>) {
    val n = arr.size
    for (i in 0 until n-1) {
        var minIdx = i
        for (j in i+1 until n) {
            if (arr[j] < arr[minIdx]) {
                minIdx = j
            }
        }
        val temp = arr[minIdx]
        arr[minIdx] = arr[i]
        arr[i] = temp
    }
}
